rpc/common/moc_common_common.proto

```diff
@@ -316,6 +317,9 @@ enum ProviderAccessOperation {
 
 	VirtualNetwork_Create           = 3400;
 	VirtualNetwork_Update           = 3401;
+
+	NetworkSecurityGroup_Create 	= 3500;
```

Comment:
How come we don't have Delete?

Reply:
I was following the operations of other SDN options in this list, I didn't see a delete for other SDN options. I can add Delete if needed for this list, but I didn't see it in many other places

Reply:
@apluchik could you check if this is correct? Don't we need `Delete`?